package/debian/rules.jammy

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/make -f
```

Comment:
this file appears identical to the existing rules file, so let's delete it and keep using that one.

Reply:
It differs in the line `export DEB_BUILD_MAINT_OPTIONS=hardening=-stackprotector optimize=-lto`; the link in the PR body gives some context that I'll add here in a comment.